cmd/event-collector/main.go

```diff
@@ -68,7 +68,7 @@ func main() {
 
 	k, err := loadConfig(cfgFile)
 	if err != nil {
-		fmt.Fprintf(os.Stderr, "Error: %v\n", err)
+		log.Logger.Error(fmt.Sprintf("Error: %v", err))
```

Comment:
Use either `Errorf` or `Errorw` depending how you want to include the error message, don't use `Sprintf` to format the message

Reply:
Also no need to have `Error:` in the message, it will be logger as error already

Reply:
this applies to the rest also

Reply:
and use lowercase messages, I see lowercase everywhere (don't capitalise)

Reply:
Hi, I tried the above suggestion but am having some problems. `log.Logger.Error()` receives a string whereas, `fmt.Errorf()` returns an `Error`. In that case, how can we use the logger?